doc/src/sphinx/develop/setup_dev.rst

```diff
@@ -23,14 +25,17 @@ The repository `clowder` containes the web frontend and is required. It should b
 
 ::
 
-  > git clone https://opensource.ncsa.illinois.edu/bitbucket/scm/cats/clowder.git
+  $ git clone https://github.com/clowder-framework/clowder
+  Depricated: https://opensource.ncsa.illinois.edu/bitbucket/scm/cats/clowder.git
 
 
 Most of the other repositories include specific extractors. Basic extractors are available in `extractors-core`:
 
 ::
 
-  > git clone https://opensource.ncsa.illinois.edu/bitbucket/scm/cats/extractors-core.git
+  $ git clone https://github.com/clowder-framework/pyclowder
+  $ git clone https://github.com/clowder-framework/extractors-core
+  Depricated: https://opensource.ncsa.illinois.edu/bitbucket/scm/cats/extractors-core.git
```

Comment:
same here, lets remove them

Reply:
Same: Per discussion on Slack with Luigi, we will remove this page and move the remaining useful bits into the "Customizing" page.